package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sphere-node-product-csv-sync",
   "description": "Import and export your products from/to your SPHERE.IO project.",
-  "version": "0.6.8",
+  "version": "0.6.11",
```

Comment:
Right i will fix that tonigjt

Sent from my iPhone

> On 05 Nov 2015, at 18:39, Hajo <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -1,7 +1,7 @@
> >  {
> >    "name": "sphere-node-product-csv-sync",
> >    "description": "Import and export your products from/to your SPHERE.IO project.",
> > -  "version": "0.6.8",
> > +  "version": "0.6.11",
> We use grunt release to bump the version - this will also publish the module on npmjs.org
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 


[image: commercetools logo] <http://www.commercetools.com/>

Amtsgericht München, HRB 161496
Geschäftsführer: Denis Werner, Dirk Hörig, Dr. Robert Zores

www.commercetools.com